doc/api/process.md

```diff
@@ -1738,7 +1738,8 @@ added:
 
 * {integer}
 
-The `process.ppid` property returns the PID of the current parent process.
+The `process.ppid` property returns the PID of the parent of the 
```

Comment:
```suggestion
The `process.ppid` property returns the PID of the parent of the
```

FYI, if you run `make lint-md` on your changes, it will report these sorts of lint issues in the .md files.

Reply:
@Trott - Thanks for the suggestion. lint issue is fixed. Please have a look.